Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1851,6 +1854,39 @@ class BrowserViewController: UIViewController, SearchBarLocationProvider, Themea
 
     // MARK: Autofill
 
+    private func creditCardInitialSetupTelemetry() {
+        // Initial telemetry for credit card autofill status
+        let userDefaults = UserDefaults.standard
+        let key = PrefsKeys.KeyAutofillCreditCardStatus
+        // Default value is true for autofill credit card input
+        let autofillStatus = userDefaults.value(forKey: key) as? Bool ?? true
+        TelemetryWrapper.recordEvent(
+            category: .action,
+            method: .tap,
+            object: .creditCardAutofillEnabled,
+            extras: [
+                TelemetryWrapper.ExtraKey.isCreditCardAutofillEnabled.rawValue: autofillStatus.description
+            ]
+        )
+
+        // Note: Added a 2 second delay so that profile is setup by the team we
+        // request for has account
+        DispatchQueue.main.asyncAfter(deadline: .now() + 2) { [weak self] in
+            guard let self = self else { return }
+            guard self.profile.hasAccount() else { return }
+            var engineEnablements = self.profile.syncManager.getEngineEnablementChangesForAccount()
```

Comment:
@lougeniaC64 I am always getting an empty list of engineEnablements?
Is this expected? 🤔

Reply:
Going to investigate, as I see tabs in that list if I go deep into the `getEngineEnablementChangesForAccount` however I am tagging you in case I am missing something here